server/resource.go

```diff
@@ -85,18 +85,48 @@ func (s *Server) registerResourceRoutes(g *echo.Group) {
 		}
 		defer src.Close()
 
-		fileBytes, err := io.ReadAll(src)
-		if err != nil {
-			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to read file").SetInternal(err)
+		var resourceCreate *api.ResourceCreate
+		systemSettingStorageServiceName := api.SystemSettingStorageServiceName
+		systemSetting, err := s.Store.FindSystemSetting(ctx, &api.SystemSettingFind{Name: &systemSettingStorageServiceName})
+		if err != nil && common.ErrorCode(err) != common.NotFound {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to find storage").SetInternal(err)
 		}
+		if common.ErrorCode(err) == common.NotFound || systemSetting.Value == "Database" {
```

Comment:
* empty string: store into database
* others: find storage table

```suggestion
		if common.ErrorCode(err) == common.NotFound || systemSetting.Value == "" {
```